flowzone.yml

```diff
@@ -3216,6 +3217,7 @@ jobs:
       - *getAWSCallerIdentity
       - *convenienceFunctions
 
+      # FIXME: https://github.com/actions/setup-python/issues/705
```

Comment:
We use setup python all over this workflow, why is this the first time we encountered issues with it on some runners?

Which specific runners are the issue? Can't we just use the python version already installed on the runner?

Reply:
Python only runs fro two types on projects: CloudFormation and Python. The former is triggered for any environment repo with AWS templates, the later, I have no idea - I don't think we have many active Python projects.

Reply:
Can we just use the existing python version on the runner to save a step and possible breakages? It's just a linter after all, it shouldn't need latest python or latest pip and those just add time and complexity.

Reply:
We can, but this is going wreck havoc with other Python dependencies (since we aren't setting up venvs). Might just work enough though.

Reply:
The runner is destroyed after the job anyway, so any havoc would be contained to the CF test job at least.